README_v2.md

```diff
@@ -0,0 +1,307 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ -->
+
+# Welcome to the Apache Solr project!
+-----------------------------------
+
+Solr is the popular, blazing fast open source enterprise search platform
+written in Java and using [Apache Lucene](https://lucene.apache.org/).
+
+For a complete description of the Solr project, team composition, source
+code repositories, and other details, please see the Solr web site at
+https://solr.apache.org/
+
+## Online Documentation
+
+This README file only contains basic instructions.  For comprehensive documentation,
+visit the [Solr Reference Guide](https://solr.apache.org/guide/).
+
+## Getting Started
+FIXME maybe put the tutorial that Noble or Ishan wrote???   Then at the end put the examples?
+
+### Starting Solr
+
+Start a Solr node in cluster mode (SolrCloud mode)
+
+```
+bin/solr -c
+```
+
+To start another Solr node and have it join the cluster alongside the first node,
+
+```
+bin/solr -c -z localhost:9983 -p 8984
+```
+
+An instance of the cluster coordination service, i.e. Zookeeper, was started on port 9983 when the first node was started. To start Zookeeper separately, please refer to XXXX.
+
+### Creating a collection
+
+Like a database system holds data in tables, Solr holds data in collections. A collection can be created as follows:
+
+```
+curl --request POST \
+--url http://localhost:8983/api/collections \
```

Comment:
[-0] If we are trying to fixup the v2 APIs before marketing them, lets unfortunately continue to use the v1 APIs until they are ready.


(If we plan on keeping the tutorial section)

Reply:
> If we are trying to fixup the v2 APIs

I guess that makes sense, unfortunately.  I'd love to get these APIs mostly locked in as soon as we can though, so that we can start pushing them.  On that note: anyone reading this should go review the proposed changes to the v2 API [here](https://docs.google.com/spreadsheets/d/1HAoBBFPpSiT8mJmgNZKkZAPwfCfPvlc08m5jz3fQBpA/edit?usp=sharing) 😛